pyproject.toml

```diff
@@ -12,6 +12,7 @@ packages = [
 python = "^3.6.1"
 flask = "^1.1.2"
 flask-sqlalchemy = "^2.4.4"
+SQLAlchemy = "~1.3.0"  # >= 1.4.0 changed a lot and breaks this project currently
```

Comment:
I updated all packages, however that broke flask-sqlalchemy for sqlalchemy >= 1.4.0 (known problem, should be fixed in upstream packages soonish). need to pin version < 1.4.0